templates/secrets/rename-these-secrets.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
This way all the developer has to do is rename the file and fill in their secrets as appropriate, instead of having to know the correct format of the secrets file